.github/workflows/make-protos.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate attestation libraries
+on:
+  push:
+    branches:
+      - "main"
+  schedule:
```

Comment:
Can you please add a comment describing the justification for needing the schedule?

Reply:
+1 to some more comments, even describing what it does a bit more would be nice.  E.g. It just sends the PR, right?  It doesn't auto-merge it?

Reply:
Added comments for the schedule and the create-pull-request step.